### PR TITLE
feat(icon): DLT-1806 accessibility-mac

### PR DESCRIPTION
**Pull request by fede-dp**
# adding icon accesibiliy-mac

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1806

## :book: Description

adding a new icon and keywords for it
## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

## :camera: Screenshots / GIFs

image

### CODE REVIEW

File: packages/dialtone-icons/src/keywords-icons.json [hunk 1]
```diff
@@ -1365,6 +1365,12 @@
       ]
     },
     "general": {
+      "accessibility-mac": [
+        "humanpictos",
+        "people",
+        "man silhouette",
+        "man silhouette"
```
Comment (iropolo) [inline packages/dialtone-icons/src/keywords-icons.json]: duplicated

File: packages/dialtone-icons/src/keywords-icons.json [hunk 2]
```diff
@@ -1365,6 +1365,12 @@
       ]
     },
     "general": {
+      "accessibility-mac": [
+        "humanpictos",
```
Comment (francisrupert) [inline packages/dialtone-icons/src/keywords-icons.json:1369]: Add
`disability, disabled, dda, inclusive`

**Comment by francisrupert** [comment]
I changed PR title to align to git commit convention.

**Review by francisrupert — APPROVED** [review]
Approved. Address comments from me and Nacho.

**Review by braddialpad — APPROVED** [review]

**Review by iropolo — APPROVED** [review]

**Comment by iropolo** [comment]
@fede-dp ping

**Pull request merged**